codalab/apps/authenz/models.py

```diff
@@ -1,5 +1,25 @@
+import os
 from django.db import models
 from django.contrib.auth import models as auth_models
+from django.utils.functional import cached_property
+from django.conf import settings
+from django.core.files.storage import get_storage_class
+
+## Needed for computation service handling
```

Comment:
I did this in the docker branch, it's available from `from apps.web.utils import PublicStorage, BundleStorage`